spanner/api/Spanner.Samples.Tests/CopyBackupWithMultiRegionEncryptionAsyncTest.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2024 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Spanner.Admin.Database.V1;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+[Collection(nameof(SpannerFixture))]
+public class CopyBackupWithMultiRegionEncryptionAsyncTest
+{
+    private readonly SpannerFixture _fixture;
+
+    public CopyBackupWithMultiRegionEncryptionAsyncTest(SpannerFixture fixture)
+    {
+        _fixture = fixture;
+    }
+
+    [SkippableFact]
+    public async Task TestCopyBackupWithMultiRegionEncryptionAsync()
+    {
+        CopyBackupWithMultiRegionEncryptionAsyncSample copyBackupSample = new CopyBackupWithMultiRegionEncryptionAsyncSample();
+        string source_Project_id = _fixture.ProjectId;
```

Comment:
Any reason these are using underscores instead of `sourceProjectId` etc?

You could potentially introduce a record type to make this simpler:

```csharp
var source = new BackupName(_fixture.ProjectId, _fixture.InstanceId, _fixture.BackupId);
var target = new BackupName(_fixture.ProjectId, _fixture.InstanceId, SpannerFixture.GenerateId("test_", 16));
...
var backup = await copyBackupSample.CopyBackupWithMultiRegionEncryptionAsync(
    source.InstanceId, source.ProjectId, source.BackupId,
    target.InstanceId, target.ProjectId, target.BackupId,
    expireTime, kmsKeyNames);
...
}

// As a nested type
public record BackupName(string ProjectId, string InstanceId, string BackupId)
```

Reply:
This is assuming there isn't a good BackupName class already, of course - I'd really have hoped there would be...

Reply:
It looks like we *do* have BackupName, because it's used later. Just use that?

Reply:
I've also fixed the code from where this was copied.